app/weddingapp/tests/test_models.py

```diff
@@ -0,0 +1,50 @@
+"""
+Tests for models
+"""
+from django.test import TestCase
+from django.contrib.auth import get_user_model
+
+
+class ModelTests(TestCase):
```

Comment:
I think we need to test the case sensitivity of the username, creating a user with uppercase letters and logging in with with lowercase, or vice versa.

Reply:
Done